sdk/java-sdk-spring/src/it/java/com/example/wiring/tracing/TracingIntegratonTest.java

```diff
@@ -50,7 +51,7 @@ public TracingIntegratonTest(ApplicationContext applicationContext) {
         super(applicationContext, config);
     }
 
-    @Test
+    @Ignore //disabled ATM for https://github.com/lightbend/kalix-jvm-sdk/pull/1810
```

Comment:
I'm confused now. 
In #1810, we added the Ignore annotation and that's what we have now in main. 

I think the diff should not even show here.

Reply:
Right. I mess the order I guess. I would have seen this if I would have made a rebase from main. Never mind.

Reply:
But that's also surprising. Ok that you needed to rebase, but GitHub compares your branch to `main` since you were planning to merge it into `main` and therefore it should not show any diff. 

Possibly, the PR was made before #1810 got merged. Still, I thought that GH would refresh the diff anyway, but I might be wrong. 🤷